base/random.jl

```diff
@@ -301,7 +323,7 @@ rand!(A::AbstractArray) = rand!(GLOBAL_RNG, A)
 rand(r::AbstractArray) = rand(GLOBAL_RNG, r)
 
 """
-    rand!([rng=GLOBAL_RNG], A, [S=eltype(A)])
+    rand!([rng=GLOBAL_RNG,] A [,S=eltype(A)])
```

Comment:
I confirm that I prefer the other syle, this `A` swandiched in between 2 brackets looks ugly.